tests/helix/send-to-helix-basictests.targets

```diff
@@ -21,10 +24,14 @@
 
     <ItemGroup>
       <_DefaultWorkItems Include="$(WorkItemArchiveWildCard)" />
+      <!-- runsettings timeout in ms, default to 10 mins -->
+      <_DefaultWorkItems TimeoutMs="600000" />
+
+      <_DefaultWorkItems Condition="'%(FileName)' == 'Aspire.Hosting.Elasticsearch.Tests'" TimeoutMs="1200000" />
```

Comment:
This is a really long time. Do we know why it is taking that long?

Reply:
I think because elasticsearch takes a few minutes to start up, and in the functional tests we are starting that up per test.

Reply:
That's what it looked like from the log. The helix agents is underpowered compared to the build machine, so it would affect it too.